wagtail-{{ cookiecutter.project_name_kebab }}/pyproject.toml

```diff
@@ -0,0 +1,68 @@
+[build-system]
+requires = ["flit_core >=3.2,<4"]
+build-backend = "flit_core.buildapi"
+
+[project]
+name = "wagtail-{{ cookiecutter.project_name_kebab }}"
+authors = [{name = "{{ cookiecutter.full_name }}", email = "{{ cookiecutter.email }}"}]
+description = "{{ cookiecutter.project_short_description }}"
+readme = "README.md"
```

Comment:
This will be used as the `long_description` and handles content type
https://flit.pypa.io/en/stable/pyproject_toml.html#new-style-metadata